index.html

```diff
@@ -355,6 +356,158 @@ <h3>Manifest Contexts</h3>
+						as JSON-LD does not currently include facilities for this. As an interim measure, this
+						specification defines a <code>direction</code> property for this purpose, as described in the
+						following table.</p>
+
+					<table class="zebra">
+						<thead>
+							<tr>
+								<th>Term</th>
+								<th>Description</th>
+								<th>Required Value</th>
+								<th>Value Category</th>
+								<th>[[!schema.org]] Mapping</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td>
+									<code data-dfn-for="PublicationManifest">
+										<dfn>direction</dfn>
+									</code>
+								</td>
+								<td>Default <a>base direction</a> for textual manifest values</td>
+								<td><code>ltr</code>, <code>rtl</code>, or <code>auto</code></td>
+								<td>
+									<a href="#value-literal">Literal</a>
+								</td>
+								<td>(None)</td>
+							</tr>
+						</tbody>
+					</table>
+
+					<p>The <dfn data-lt="TextDirection">base language direction</dfn> MUST have one of the following
+						values:</p>
+
+					<div data-dfn-for="TextDirection">
+						<ul data-include="common/html/dir-values.html" data-include-replace="true"></ul>
+					</div>
+
+					<p>No default values are specified for the global language or base direction.</p>
+				</section>
+
+				<section id="manifest-lang-dir-local">
+					<h4>Item-Specific Declarations</h4>
+
+					<p>It is possible to set the language locally for any textual value in the manifest using a <a
+							href="#LocalizableString">localizable string</a>:</p>
+
+					<pre class="example" title="Setting the author name to French using a localizable string">
+{
+    "@context" : ["https://schema.org","https://www.w3.org/ns/pub-context"],
+    "type"     : "Book",
+    &#8230;
+    "author" : {
+        "type"  : "Person",
+        "name" : {
+            "value"    : "Marcel Proust",
+            "language" : "fr"
+        }
+    }
+}
+</pre>
+					<p>The value of the <code>language</code> MUST be set to a language code as defined in
+						[[!bcp47]].</p>
+
+					<p>When used in a context of localizable texts, a simple string value is a shorthand for a
+							<a>localizable string</a>, with the <code>value</code> set to the string value, and the
+						language set to the value of the <a href="#manifest-default-lang-dir-default">default
+								<code>language</code> property</a>, if applicable, and unset otherwise. In other words,
+						the previous example is equivalent to:</p>
+
+					<pre class="example" title="Setting the default language of an author name to French">
+{
+    "@context"   : [
+        "https://schema.org",
+        "https://www.w3.org/ns/pub-context",
+        {"language": "fr"}
+    ],
+    "type"       : "Book",
+    &#8230;
+	"author"     : "Marcel Proust",
+    &#8230;	
+}
+</pre>
+					<p>See <a href="#value-objects"></a> for more information.</p>
+
+					<p>It is <em>not</em> possible to set the direction explicitly for a specific value.</p>
+
```

Comment:
I would think (reflecting to one of your questions in the preamble) that this is a good place for the reference to string-meta